ide/wg_Completion.ml

```diff
@@ -8,17 +8,7 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
-module StringOrd =
-struct
-  type t = string
-  let compare s1 s2 =
-    (* we use first size, then usual comparison *)
-    let d = String.length s1 - String.length s2 in
-    if d <> 0 then d
-    else compare s1 s2
-end
-
-module Proposals = Set.Make(StringOrd)
+module Proposals = CString.Set
```

Comment:
Sorry to only wake up after the merge, but *as duly explained by the comment* this is wrong. Proposals need to be ordered by length first, for UI reasons.

Reply:
See #11965.

Reply:
Ups, sorry, fix merged.